test/jsonapi/view_test.exs

```diff
@@ -146,10 +146,17 @@ defmodule JSONAPI.ViewTest do
   end
 
   test "url_for_pagination/3" do
-    assert PostView.url_for_pagination(nil, nil, %{}) == "/api/posts"
+    conn = Plug.Conn.fetch_query_params(%Plug.Conn{})
```

Comment:
What do you think about pulling these changes out into their own tests, specifically testing the retention of query params on pagination links?